remo/profiles/tests/test_views.py

```diff
@@ -204,14 +204,6 @@ def test_number_of_reps_visibility_unauthed(self):
         d = pq(response.content)
         eq_(len(d('#profiles-number-of-reps')), 0)
 
-    def test_number_of_reps_visibility_authenticated(self):
```

Comment:
I think you remove the wrong test here. The test to remove is the one above this one.